javascript-if/main.js

```diff
@@ -0,0 +1,65 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+*/
+
+function isUnderFive(number){
+  var statement = number < 5
+    return statement;
+}
+
+function isEven(number){
+  var statment = number % 2 === 0
+  return statment;
+}
+
+function startsWithJ(string){
+  var statement = string[0] === "J"
+  return statement;
+}
+
+function isOldEnoughToDrink(person){
+  var statement = person.age >= 21
+  return statement;
+}
+
+function isOldEnoughToDrive(person){
+  var statement = person.age >= 16
+  return statement;
+}
+
+function isOldEnoughToDrinkAndDrive(person){
+  var statement = person.name === " "
+  return statement;
```

Comment:
You should just return `false` for this one, this will probably lead to false every time, but if you get a driver that is named " ", they're apparently now allowed to drive drunk!